### DISCUSSION TITLE
NTV

### DISCUSSION THREAD

**Discussion by Niyasmkkv** [iptv-org/community#688] [Q&A]
**Channel Name:** NTV UAE
**Channel ID:** NTV.ae
**Country:** UAE
**Language:** Malayalam
**Website:** http://www.ntvuae.tv/
**Stream URL (optional):** 
**Notes (optional):** xxx

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19725
